signals/generators/ios/templates/data_model.m.j2

```diff
@@ -27,11 +27,8 @@
 
 - (void) setup {
   // Initialize RestKit
-  {% if is_url(api_url) %}
-  NSURL *baseURL = [NSURL URLWithString:@"{{ api_url }}"];
-  {% else %}
-  NSURL *baseURL = {{ api_url }};
-  {% endif %}
+  NSAssert(_delegate != NULL, @"Set DataModel.sharedDataModel().delegate before calling DataModel.sharedDataModel().setup()");
```

Comment:
On second thought, I think we should just pass the delegate into the setup function.  Anyone disagree?